postgres/src/test/scala/zio/sql/postgresql/FunctionDefSpec.scala

```diff
@@ -153,6 +153,21 @@ object FunctionDefSpec extends PostgresRunnableSpec with ShopSchema {
           r <- testResult.runCollect
         } yield assert(r.head)(equalTo(expected))
 
+        assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
+      },
+      testM("to_ascii") {
```

Comment:
I can't write a success case test because our test DB default encoding is UTF-8 but `to_ascii("hello")` fails with the following error:
```
org.postgresql.util.PSQLException: ERROR: encoding conversion from UTF8 to ASCII not supported
```

From postgresql docs:
```
Conversion is only supported from LATIN1, LATIN2, LATIN9, and WIN1250 encodings
```

Reply:
Right, sorry I misread your original comment. I'll add this to another existing issue so we can get you a shiny new container with the right encoding for this test, but this will do for now.